src/material-examples/autocomplete-display/autocomplete-display-example.css

```diff
@@ -0,0 +1,7 @@
+.example-form {
+  width: 500px;
+}
```

Comment:
@willshowell - I was thinking about going through and removing all these fixed widths since it causes responsive issues. I'd like to do that in a separate PR.